mantidproject/mantid
# Indirect - Interface plot error bars

In IndirectTab, all plotting functions now have error bars enabled as standard, so any plotting initiated from Indirect Inelastic interfaces should display error bars 

**To test:**
Run through a couple of indirect interfaces and ensure a graph with error bars is displayed when `plot` is clicked.
For Example:
Instrument: IRIS
Indirect > Data Reduction > Calibration
Run No: `26173`
Indirect > Data Reduction > Energy Transfer
Run No: `26176`
Use `26173_calib` workspace as calibration

Indirect > Data Analysis > Elwin
Use `26176_red` workspace as input
Indirect > Data Analysis > MSDFit
Use `26176_elwin_eq2` workspace as input

Fixes #17188.

[Release Notes](https://github.com/mantidproject/mantid/blob/b3c69f08b29aee3dacb7d3db053806ef39d9c7d7/docs/source/release/v3.9.0/indirect_inelastic.rst)

---
#### Reviewer

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):
##### Code Review
##### Functional Tests
- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
